ext/session/session.c

```diff
@@ -2113,7 +2113,7 @@ PHP_FUNCTION(session_set_save_handler)
 		RETURN_TRUE;
 	}
 
-	php_error_docref(NULL, E_DEPRECATED, "Calling session_set_save_handler() with more than 2 arguments is deprecated");
+	php_error_docref(NULL, E_DEPRECATED, "providing individual callbacks instead of an object implementing SessionHandlerInterface is deprecated");
```

Comment:
AFAIR we settled on using uppercase initials in error messages back then with Nikita.
```suggestion
	php_error_docref(NULL, E_DEPRECATED, "Providing individual callbacks instead of an object implementing SessionHandlerInterface is deprecated");
```

Reply:
only the tests should be adapted now